src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-manager-message.service.ts

```diff
@@ -56,81 +61,104 @@ export class TaskManagerMessageService {
     ),
     'rbd/clone': new TaskManagerMessage(
       (metadata) => `Clone RBD '${metadata.child_pool_name}/${metadata.child_image_name}'`,
-      (metadata) => `RBD '${metadata.child_pool_name}/${metadata.child_image_name}'
-                     has been cloned successfully`,
+      (metadata) => `Cloning RBD '${metadata.child_pool_name}/${metadata.child_image_name}'`,
+      (metadata) =>
+        `RBD '${metadata.child_pool_name}/${
+          metadata.child_image_name
+        }' has been cloned successfully`,
       (metadata) => {
         return {
-          '17': `Name '${metadata.child_pool_name}/${metadata.child_image_name}' is already
-                 in use.`,
+          '17': `Name '${metadata.child_pool_name}/${
+            metadata.child_image_name
+          }' is already in use.`,
           '22': `Snapshot must be protected.`
         };
       }
     ),
     'rbd/copy': new TaskManagerMessage(
       (metadata) => `Copy RBD '${metadata.dest_pool_name}/${metadata.dest_image_name}'`,
-      (metadata) => `RBD '${metadata.dest_pool_name}/${metadata.dest_image_name}'
-                     has been copied successfully`,
+      (metadata) => `Coping RBD '${metadata.dest_pool_name}/${metadata.dest_image_name}'`,
```

Comment:
Both `title`and `running` message start with similar words:

![screenshot from 2018-06-05 17-48-03](https://user-images.githubusercontent.com/14297426/40990477-dd34a9a4-68e8-11e8-92c3-953f7a6f85d4.png)

Personally, I prefer the "language" we had before:

![screenshot from 2018-06-05 17-47-24](https://user-images.githubusercontent.com/14297426/40990504-f35f630e-68e8-11e8-98d2-ac5022cdac9a.png)

But this is just a personal preference.

Reply:
Typo - s/Coping/Copying/

Reply:
Now the behavior is different - You will only see `Coping RBD 'rbd/i5'` as running task.
But you are showing the Notification tab here - as I mentioned I wanted to change the behavior in another PR, in order to have similar messages.

Reply:
Fixed typo :)